src/en/03_user_guide.md

```diff
@@ -186,4 +186,38 @@ Once the desired information has been filled out, the query may be executed with
 
 ![Image showing the generated template with example data](./images/template2.png)
 
-> **Note:** In future releases, this feature is planned to be re-implemented in an user interface.
\ No newline at end of file
+> **Note:** In future releases, this feature is planned to be re-implemented in an user interface.
+
+---
+
+## Backup and Restore
+
+The **Backup** and **Restore** features provide a user interface for PostgreSQL’s [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html), [pg_dumpall](https://www.postgresql.org/docs/current/app-pg-dumpall.html), and [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) commands. PgManage allows you to create backups for a database or the whole server. The backups can be a `.tar` or a `directory`. 
+
+The different actions will be displayed under the `Jobs` section. Here, information such as PID, Type, Server, Object, Start Time, Status, Duration, and Actions will be displayed.
```

Comment:
Perhaps we should change "actions" here to something like backup/restore jobs to avoid confusion with the "Actions" column mentioned below?

These elements with pid, type etc - display a specific backup/restore job properties